src/main/scala/org/renci/cam/Biolink.scala

```diff
@@ -25,9 +25,9 @@ object Biolink {
     for {
       (biolinkPrefixes, classes, predicates) <- getBiolinkPrefixesAndClassesAndPredicatesFromFile
       local <- localPrefixes
-//      prefixes <- getBiolinkPrefixesFromFile
-      prefixes <- getBiolinkPrefixesFromURL.orElse(getBiolinkPrefixesFromFile)
-      combined = local ++ biolinkPrefixes ++ prefixes
+      prefixesFromFile <- getBiolinkPrefixesFromFile
+      prefixesFromURL <- getBiolinkPrefixesFromURL
+      combined = local ++ biolinkPrefixes ++ prefixesFromFile ++ prefixesFromURL
```

Comment:
I think we've kind of lost the purpose of all these variants. The local file (currently "legacy_prefixes.json", could be renamed) is prefixes for namespaces we use but aren't in Biolink. The data in "prefixes.json" is just meant to be a fallback recent Biolink context in case the Biolink file isn't reachable for some reason. And I'm not sure exactly why we are taking the prefixes out of the Biolink model (biolinkPrefixes). I would expect these all to be in the Biolink context.

Reply:
Not sure I totally agree with this sentiment.  Yes, "prefixes.json" is a fallback for when https://biolink.github.io/biolink-model/context.jsonld can't be reached or if formatting changes were made (which happened a day or so ago).  Yes, "legacy_prefixes.json" is for prefixes not found in https://biolink.github.io/biolink-model/context.jsonld.  Yet, the prefixes from the https://biolink.github.io/biolink-model/biolink-model.yaml once included some mappings that were not found elsewhere.  Granted, the https://biolink.github.io/biolink-model/context.jsonld file seems to be changing readily & it might be that parsing https://biolink.github.io/biolink-model/biolink-model.yaml for prefixes is no longer necessary, however, it's an easy grab since we already parse it for classes & predicates.